src/components/WeatherPanel.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-throw-literal */
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react';
+import Form from './Form';
+import Card from './Card';
+
+const WeatherPanel = () => {
+  let urlWeather = 'https://api.openweathermap.org/data/2.5/weather?&appid=56cdd6a9ec3b0492107abe99f711937a&lang=EN';
```

Comment:
- [ ] Nice working on your components, kindly check I get the below errors when I try to search or load the project.

![error---](https://user-images.githubusercontent.com/48371156/190881905-e2e6f502-0b95-4291-b178-143cd0391f2a.png)